### ISSUE TITLE
Unable to remove invalid image-src CSP policy

### ISSUE THREAD

**Issue by GeeWee** [stripe/stripe-apps#578]
**Describe the bug**
Previously we had an invalid image-src CSP with the path `/docs/_next/image?url=%2Fdocs%2Flogo.png&w=128&q=75'`

We get errors in Chrome mentioning that this path is invalid (which it is)

![image](https://user-images.githubusercontent.com/6381792/168810096-a9930ef7-2969-41b6-a1d8-2e9ab7ec8086.png)

**However**, we are unable to remove the image-src CSP.
Currently our manifest, which I have tried to run both using `stripe apps start` and `stripe apps upload` multiple times looks like this:

```json
    "content_security_policy": {
      "connect-src": [
        "...some-urls"
      ],
      "image-src": null,
      "purpose": "These links allow us to contact our backend services to make carbon estimations"
    }
```

I have also tried turning `image-src` into an empty array (`"image-src": []`) and no matter what, the image-src CSP is not updated on Stripe's end.

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] - Fedora but can reproduce on MacOS too
 - Browser [e.g. chrome, safari] - Reproducible on both Firefox and Chrome

**Comment by cchalstrom-stripe** [comment]
@GeeWee and just to make sure, when you try to upload a new version with the fixed CSP, are you also incrementing the `version` of your stripe-app.json?

**Comment by GeeWee** [comment]
Yeah! I've tried both with `stripe apps start`, `stripe apps upload` (plus patch version bump) and both `[]` and `null` as the `image-src`
